packages/admin/admin/src/table/ExcelExportButton.tsx

```diff
@@ -1,7 +1,8 @@
-import { Button, CircularProgress } from "@mui/material";
+import { CircularProgress } from "@mui/material";
 import { PropsWithChildren, ReactNode } from "react";
 import { FormattedMessage } from "react-intl";
 
+import { Button } from "../common/buttons/Button";
```

Comment:
This `Button` needs `variant="textDark"`, right? 
Otherwise it will be the new `primary` variant.